util/expensivequery/expensivequery.go

```diff
@@ -73,6 +75,9 @@ func (eqh *Handle) Run() {
 				}
 			}
 			threshold = atomic.LoadUint64(&variable.ExpensiveQueryTimeThreshold)
+			if record.err == nil {
+				record.alarm4ExcessiveMemUsage(sm, ctx)
```

Comment:
Is it too frequent to record memory usage once 100ms?

Reply:
It only checks memory usage at this frequency. The interval between twice times recording is at least 10 second.

Reply:
OK.